virt/driver.go

```diff
@@ -555,7 +555,7 @@ func (d *VirtDriverPlugin) StartTask(cfg *drivers.TaskConfig) (*drivers.TaskHand
 		return nil, nil, fmt.Errorf("virt: failed to decode driver config: %v", err)
 	}
 
-	d.logger.Debug("starting task", "driver_cfg", hclog.Fmt("%+v", driverConfig))
+	d.logger.Error("starting task", "driver_cfg", hclog.Fmt("%+v\n", driverConfig))
```

Comment:
Is this a debugging change? If not, I don't think this should be an error message.